aspnetcore/migration/60-70.md

```diff
@@ -71,15 +71,29 @@ In the project file, update each [Microsoft.AspNetCore.*](https://www.nuget.org/
 </ItemGroup>
 ```
 
-<!--
 ## Blazor
 
-After following the guidance earlier in this article to update an app to 7.0, adopt specific features by following the links in <xref:aspnetcore-7.0#blazor>.
+### Adopt .NET 7 features
+
+After following the guidance earlier in this article to update an app to 7.0, adopt specific features by following the links in <xref:aspnetcore-7#blazor>.
 
-To adopt all of the [new 6.0 features for Blazor apps](xref:aspnetcore-7.0#blazor), we recommend the following process:
+To adopt all of the [new 7.0 features for Blazor apps](xref:aspnetcore-7#blazor), we recommend the following process:
 
 * Create a new 7.0 Blazor project from one of the Blazor project templates. For more information, see <xref:blazor/tooling>.
```

Comment:
I don't think we want to recommend creating a new project to move to .NET 7. You should be able to update a .NET 6 project to .NET 7 in-place

Reply:
It's expensive 💰 ... describing every template change step-by-step? I did that once before. It took hours to write out.